.gitpod.yml

```diff
@@ -0,0 +1,20 @@
+image: nfcore/gitpod:latest
+tasks:
+  - name: install current state of nf-core/tools
+    command: |
+      NFCORE_BUILD_PATH=$HOME/nf-core_build
+      pip install --upgrade -r requirements-dev.txt -e . -t $NFCORE_BUILD_PATH
+      PATH="$NFCORE_BUILD_PATH:$PATH"
+vscode:
+  extensions: # based on nf-core.nf-core-extensionpack
+    - codezombiech.gitignore                 # Language support for .gitignore files
+    # - cssho.vscode-svgviewer                 # SVG viewer
+    - davidanson.vscode-markdownlint         # Markdown/CommonMark linting and style checking for Visual Studio Code
+    - eamodio.gitlens                        # Quickly glimpse into whom, why, and when a line or code block was changed
+    - EditorConfig.EditorConfig              # override user/workspace settings with settings found in .editorconfig files
+    - Gruntfuggly.todo-tree                  # Display TODO and FIXME in a tree view in the activity bar
+    - mechatroner.rainbow-csv                # Highlight columns in csv files in different colors
+    # - nextflow.nextflow                      # Nextflow syntax highlighting
+    - oderwat.indent-rainbow                 # Highlight indentation level
+    - redhat.vscode-yaml                     # YAML Language support
```

Comment:
```suggestion
```
It looks like gitpod installs this package already globally, so we can remove it from the list and get rid of the non-useful linting message: "redhat.vscode-yaml extension is not installed, but not removed from .gitpod.yml".